app/.idea/workspace.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ChangeListManager">
+    <list default="true" id="1f61e017-05a1-40a9-a852-d943b0f421f0" name="Default Changelist" comment="">
+      <change beforePath="$PROJECT_DIR$/src/main/java/com/keylesspalace/tusky/db/AccountDao.kt" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/com/keylesspalace/tusky/db/AccountDao.kt" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/main/java/com/keylesspalace/tusky/db/AccountManager.kt" beforeDir="false" afterPath="$PROJECT_DIR$/src/main/java/com/keylesspalace/tusky/db/AccountManager.kt" afterDir="false" />
+    </list>
+    <option name="SHOW_DIALOG" value="false" />
+    <option name="HIGHLIGHT_CONFLICTS" value="true" />
+    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
+    <option name="LAST_RESOLUTION" value="IGNORE" />
+  </component>
+  <component name="Git.Settings">
+    <option name="RECENT_GIT_ROOT_PATH" value="$PROJECT_DIR$/.." />
+  </component>
+  <component name="ProjectId" id="1khX1EAPmH4WViEc7LF6Ngx8d5R" />
+  <component name="ProjectLevelVcsManager" settingsEditedManually="true" />
+  <component name="ProjectViewState">
+    <option name="hideEmptyMiddlePackages" value="true" />
+    <option name="showLibraryContents" value="true" />
+  </component>
+  <component name="PropertiesComponent">
+    <property name="RunOnceActivity.OpenProjectViewOnStart" value="true" />
+    <property name="RunOnceActivity.ShowReadmeOnStart" value="true" />
+    <property name="android.sdk.path" value="$USER_HOME$/.android" />
+  </component>
+  <component name="SvnConfiguration">
+    <configuration />
+  </component>
+  <component name="TaskManager">
+    <task active="true" id="Default" summary="Default task">
+      <changelist id="1f61e017-05a1-40a9-a852-d943b0f421f0" name="Default Changelist" comment="" />
+      <created>1606153579073</created>
+      <option name="number" value="Default" />
+      <option name="presentableId" value="Default" />
+      <updated>1606153579073</updated>
+    </task>
+    <servers />
+  </component>
+</project>
```

Comment:
I don't think this file should be checked in

Reply:
nope, was a mistake, slipped in at some point, sorry!